mne/io/edf/tests/test_gdf.py

```diff
@@ -38,15 +38,15 @@ def test_gdf_data():
     # Assert data are almost equal
     assert_array_almost_equal(data, data_biosig, 8)
 
-    # Test for stim channel
-    events = find_events(raw, shortest_event=1)
-    # The events are overlapping.
-    assert_array_equal(events[:, 0], raw._raw_extras[0]['events'][1][::2])
+    # # Test for stim channel
+    # events = find_events(raw, shortest_event=1)
+    # # The events are overlapping.
+    # assert_array_equal(events[:, 0], raw._raw_extras[0]['events'][1][::2])
```

Comment:
This shows, that actually there are events in `gdf1_path` and these should be their onsets:
```py
ipdb> raw._raw_extras[0]['events'][1][::2]                                    
array([ 5127,  5697,  5728,  5850,  5936,  5957,  5992,  6059,  6084,
        6132,  6165,  6256,  6389,  6423,  6459,  6572,  6657,  6706,
        6740,  6805,  6833,  6857,  6951,  6982,  7008,  7043,  7139,
        7205,  7314,  7434,  7474,  7540,  7607,  7636,  7660,  7686,
        7788,  7844,  7879,  7911,  8050,  8089,  8128,  8196,  8257,
        8281,  8308,  8349,  8373,  8415,  8458,  8496,  8582,  8620,
        8644,  8711,  8737,  8780,  8904,  8935,  8979,  9014,  9107,
        9152,  9247,  9275,  9392,  9457,  9512,  9556,  9667,  9729,
        9755,  9853,  9939,  9968, 10045, 10153, 10290, 10357, 10413,
       10504, 10567, 10642, 10666, 10708, 10738, 10845, 10896, 10945,
       11042, 11109, 11220, 11246, 11291, 11335, 11385, 11931, 11974,
       12026, 12093, 12168, 12199, 12264, 12346, 12398, 12515, 12564,
       12590, 12625, 12738, 12762, 12803, 12848, 12873, 12939, 12980,
       13010, 13075, 13147, 13186, 13219, 13247, 13298, 13328, 13355,
       13400, 13466, 13497, 13569, 13594, 13656, 13727, 13764, 13798,
       13821, 13844, 13896, 13946, 14012, 14087, 14198, 14335, 14384,
       14458, 14509, 14531, 14556, 14614, 14673, 14752, 14819, 14854,
       14877, 14930, 14956, 14989, 15009, 15052, 15142, 15186, 15207,
       15287, 15354, 15386, 15416, 15453, 15488, 15589, 15675, 15760,
       15785, 15860, 15890, 15918, 15984, 16014, 16086, 16203, 16280,
       16366, 16458, 16486, 16574, 16602, 16721, 16746, 16812, 16888,
       16920, 17009, 17058, 17126, 17673, 17752, 17776, 17810, 17944,
       18034, 18106, 18138, 18212, 18236, 18358, 18388, 18418, 18448,
       18474, 18528, 18552, 18582, 18655, 18698, 18732, 18813, 18898,
       18930, 18955, 19062, 19115, 19200, 19334, 19363, 19430, 19484,
       19516, 19551, 19589, 19616, 19649, 19726, 19752, 19796, 19922,
       19950, 20024, 20045, 20091, 20212, 20240, 20295, 20319, 20350,
       20393, 20487, 20594, 20647, 20715, 20748, 20776, 20845, 20878,
       20912, 20978, 21010, 21048, 21152, 21187, 21212, 21279, 21304,
       21336, 21366, 21428, 21468, 21552, 21658, 21733, 21772, 21869,
       21895, 21960, 21985, 22018, 22087, 22156, 22178, 22208, 22259,
       22307, 22447, 22472, 22512, 22578, 22612, 22658, 22741, 22817,
       22838, 23388, 23416, 23491, 23523, 23546, 23657, 23724, 23850,
       23968, 24060, 24178, 24314, 24397, 24468, 24493, 24529, 24557,
       24585, 24654, 24677, 24739, 24847, 24876, 24927, 24957, 24984,
       25027, 25072, 25131, 25172, 25239, 25272, 25386, 25470, 25581,
       25618, 25645, 25667, 25703, 25752, 25782, 25810, 25876, 25908,
       25954, 26032, 26076, 26098, 26160, 26185, 26304, 26334, 26386,
       26411, 26478, 26530, 26570, 26618, 26653, 26716, 26748, 26877,
       26906, 26985, 27032, 27057, 27125, 27263, 27293, 27420, 27476,
       27505, 27544, 27565, 27596, 27628, 27650, 27692, 27757, 27854,
       27962, 28058, 28080, 28141, 28165, 28311, 28333, 28374, 28477,
       28511, 28576, 28608, 28720, 28785, 28868, 28913, 29462, 29489,
       29579, 29698, 29767, 29790, 29862, 29946, 30009, 30033, 30062,
       30108, 30176, 30213, 30236, 30261, 30304, 30325, 30392, 30413,
       30479, 30535, 30563, 30585, 30629, 30690, 30718, 30786, 30844,
       30974, 31042, 31079, 31161, 31215, 31236, 31270, 31292, 31367,
       31392, 31420, 31454, 31478, 31546, 31621, 31752, 31824, 31867,
       31907, 31932, 31975, 32024, 32054, 32128, 32157, 32223, 32250,
       32286, 32312, 32348, 32442, 32519, 32587, 32610, 32634, 32705,
       32798, 32864, 32907, 32993, 33037, 33146, 33180, 33248, 33308,
       33333, 33363, 33476, 33500, 33564, 33591, 33637, 33741, 33813,
       33841, 33990, 34101, 34176, 34207, 34298, 34323, 34398, 34455,
       34483, 34534, 34565, 34609, 34880], dtype=uint32)
```

however, I cannot see to which channel these events should be encoded:
```py
ipdb> raw._raw_extras[0]['ch_names']                                          
['FP1', 'FP2', 'F5', 'AFz', 'F6', 'T7', 'Cz', 'T8', 'P7', 'P3', 'Pz', 'P4', 'P8', 'O1', 'Oz', 'O2']
```
Any idea ??

Reply:
#4205 cc: @nbara

Reply:
what do we do with this?

Reply:
@massich The file in question was provided by @alexandrebarachant in https://github.com/mne-tools/mne-python/pull/2136, I am not sure what events it **should** contain beyond those that were found in the unit test previously. 

 > however, I cannot see to which channel these events should be encoded

The GDF format supports passing events as annotations without a physical stim/trigger/status channel, so that could explain why you find events in the file.

Reply:
Yes, events in GDF are usually stored in a separate event table and not in a channel.